Src/Shared/AssertionExtensions.cs

```diff
@@ -617,20 +616,21 @@ public static PropertyInfoSelectorAssertions Should(this PropertyInfoSelector pr
         /// <exception cref = "ArgumentNullException">Thrown if eventSource is Null.</exception>
         public static void MonitorEvents(this object eventSource)
         {
+            // SMELL: This static stuff needs to go at the some point. 
             EventMonitor.AddRecordersFor(eventSource, BuildRecorders);
         }
 
         private static EventRecorder[] BuildRecorders(object eventSource)
         {
-            var recorders =
-                eventSource.GetType()
-                    .GetEvents()
-                    .Select(@event => CreateEventHandler(eventSource, @event)).ToArray();
+            EventRecorder[] recorders = eventSource
+                .GetType()
+                .GetEvents()
+                .Select(@event => CreateEventHandler(eventSource, @event)).ToArray();
 
             if (!recorders.Any())
             {
                 throw new InvalidOperationException(
-                    string.Format("Type {0} does not expose any events.", eventSource.GetType().Name));
+                    $"Type {eventSource.GetType().Name} does not expose any events.");
```

Comment:
This is a C# 6 feature, I'm not sure you want use it since all of the project doesn't use it's features (yet)